2/contrib/jenkins/kube-slave-common.sh

```diff
@@ -74,6 +74,11 @@ function generate_kubernetes_config() {
     [ -z "$oc_cmd" ] && return
     [ ! has_service_account ] && return
     local crt_contents=$(cat "${KUBE_CA}")
+    # do not dump contents of cert here, but allow any error messages to be seen
+    openssl x509 -in $KUBE_CA > /dev/null
+    if [ $? -eq 1 ] ; then
+      crt_contents=""
```

Comment:
i take it setting an empty cert in the xml is not a problem?  if so, lgtm.

Reply:
yep good question @bparees ....,I have verified by setting the cert field to empty and then restarted Jenkins.  It came up fine.  No exceptions.